src/utils.js

```diff
@@ -183,7 +183,7 @@ nv.utils.calcApproxTextWidth = function (svgTextElem) {
     if (nv.utils.isFunction(svgTextElem.style) && nv.utils.isFunction(svgTextElem.text)) {
         var fontSize = parseInt(svgTextElem.style("font-size").replace("px",""), 10);
         var textLength = svgTextElem.text().length;
-        return textLength * fontSize * 0.5;
+        return !isNaN(fontSize) ? textLength * fontSize * 0.5 : 0;
```

Comment:
Use this helper function from the same file please:

https://github.com/mode/nvd3/blob/master/src/utils.js#L195